SPECS/PyPAM/PyPAM.spec

```diff
@@ -20,86 +17,65 @@ Patch3:         PyPAM-0.5.0-memory-errors.patch
 Patch4:         PyPAM-0.5.0-return-value.patch
 Patch5:         PyPAM-python3-support.patch
 
-BuildRequires:  python2-devel
-BuildRequires:  python2-libs
-BuildRequires:  python-setuptools
-BuildRequires:  pam-devel
-BuildRequires:  python3
-BuildRequires:  python3-devel
-BuildRequires:  python3-libs
-
-Requires:       python2
-Requires:       python2-libs
-
 %description
 Python bindings for PAM (Pluggable Authentication Modules).
 
 %package -n     python3-PyPAM
```

Comment:
Since we can make breaking changes in 2.0, I say we just make `PyPAM` be the Python 3 version now, so we can drop the `python3-PyPAM` naming. This will also make image definitions migration easier between Mariner version.

Reply:
I'll defer to your judgement here- I'm not too familiar with what image definition migrations will look like.